data/strings/types_strings.txt

```diff
@@ -9193,6 +9193,318 @@
     zh-Hans = 物理治疗师
     zh-Hant = 物理治療師
 
+  [type.healthcare.alternative]
+    en = Alternative medicine
+    ar = الطب البديل
+    be = Нетрадыцыйная медыцына
+    bg = Алтернативна медицина
+    cs = Alternativní medicína
+    da = Alternativ medicin
+    de = Alternative Medizin
+    el = Εναλλακτική ιατρική
+    es = Medicina alternativa
+    et = Alternatiivne meditsiin
+    eu = Medikuntza alternatiboa
+    fa = طب جایگزین
+    fi = Vaihtoehtoinen lääketiede
+    fr = Médecine alternative
+    he = רפואה אלטרנטיבית
+    hu = Alternatív gyógyászat
+    id = Pengobatan alternatif
+    it = Medicina alternativa
+    ja = 代替医療
+    ko = 대체 약품
+    mr = पर्यायी औषध
+    nb = Alternativ medisin
+    nl = Alternatieve geneeskunde
+    pl = Medycyna alternatywna
+    pt = Medicina alternativa
+    pt-BR = Medicina alternativa
+    ro = Medicină alternativă
+    ru = Альтернативная медицина
+    sk = Alternatívna medicína
+    sv = Alternativ medicin
+    sw = Dawa mbadala
+    th = การแพทย์ทางเลือก
+    tr = Alternatif tıp
+    uk = Нетрадиційна медицина
+    vi = Liều thuốc thay thế
+    zh-Hans = 替代药物
+    zh-Hant = 替代藥物
+
+  [type.healthcare.audiologist]
+    en = Audiologist
+    ar = السمعيات
+    be = Аўдыёлаг
+    bg = Аудиолог
+    cs = Audiolog
+    da = Audiolog
+    de = Audiologe
+    el = Ακουολόγος
+    es = Audiólogo
+    et = Audioloog
+    eu = Audiologoa
+    fa = شنوایی شناس
+    fi = Audiologi
+    fr = Audiologiste
+    he = אודיולוג
+    hu = Audiológus
+    id = Audiologis
+    it = Audiologo
+    ja = オーディオロジスト
+    ko = 청력학자
+    mr = ऑडिओलॉजिस्ट
+    nb = Audiolog
+    nl = Audioloog
+    pl = Audiolog
+    pt = Audiologista
+    pt-BR = Audiologista
```

Comment:
pt-BR = Fonoaudiólogo

Reply:
Thanks!
Done in 5d32dd98de7b8591e4719e2842b617d50026572f